### PR TITLE
Bump version of dependencies

### PR DESCRIPTION
**Pull request by PatWie**
An attempt to bring dependencies up2date.

### CODE REVIEW

File: service/docker.go [hunk 1]
```diff
@@ -149,7 +150,15 @@ func (ds *DockerService) Run(
 		},
 	}
 
-	resp, err := ds.Client.ContainerCreate(ctx, cfg, hostCfg, nil, "")
+	networkConfig := &network.NetworkingConfig{
```
Comment (PatWie) [inline service/docker.go]: Unsure if this would be simply nil

**Comment by PatWie** [comment]
@keyraphi 
tested the submission locally (including the docker run) with the python example from the documentation.
The following things work:
- unit-tests from ci

manually tested:
- register 2 users (admin, student)
- create task/exercise/course
- create submission
- submit template/solution and verify output is correct (started worker in background)

I think it is safe to merge.

File: service/docker.go (hunk 1 above)
Comment (PatWie) [reply service/docker.go]: Can be simply nil

**Review by keyraphi — APPROVED** [review]
Looks good to me. Mostly simple api changes.

**Pull request merged**